.github/workflows/ubuntu-21.04.yml

```diff
@@ -0,0 +1,55 @@
+name: Ubuntu 21.04 CI
+
+on: [push, workflow_dispatch]
+
+jobs:
+  build-devenv:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image
+      run: sudo DOCKER_BUILDKIT=1 docker build ./docker/devenv --file ./docker/devenv/Dockerfile.ubuntu.2104 --tag openage-devenv:latest
+      shell: bash
+    - name: Save the Docker image
+      run: |
+        mkdir -p /tmp/staging
+        sudo docker save openage-devenv:latest | gzip > /tmp/staging/devenv.tar.gz
+      shell: bash
+    - name: Publish the Docker image
+      uses: actions/upload-artifact@v2.2.4
+      with:
+        name: devenv-image-compressed
+        path: '/tmp/staging/devenv.tar.gz'
+        if-no-files-found: error
+        retention-days: 30
+
+  build:
+    runs-on: ubuntu-20.04
+    needs: build-devenv
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
does it always run? since kevin does the checks for pull requests, is there a way to let github-actions only be run in forked child repos?

Reply:
Currently it runs on every push, and can also be manually triggered (`[push, workflow_dispatch]` on line 3).

Reply:
I believe there is a way to trigger it on specific repos, but personally I think it's also beneficial on the main repo. Kevin CI can still serve as PR checks, while this one can be the source of truth for Ubuntu building. With this workflow, we no longer need to answer "can't compile on my Ubuntu" questions. Everyone who wants to contribute can download the docker image and there's no need to setup local dev environment.
For openage users -- I know the game isn't complete yet, but one day we'll get there -- they can download the output of the second stage, which should be a package (.deb ?) that is ready for the end users. However, currently I just exported the build folder as the output for the second stage. There's some future work on packaging.

Reply:
If it's

`on: [push, pull_request]`

it should only run on forked child repos when they make a PR.